### ISSUE TITLE
[BUG] Regression - NVTabular 0.5.3 can't parse S3 paths properly.

### ISSUE THREAD

**Issue by Arnie0426** [NVIDIA/NVTabular#964]
**Describe the bug**
NVTabular 0.5.3 doesn't seem to be able to parse S3 paths properly and throws a FileNotFound Error. 
NVTabular 0.5.2 works fine though.

**Steps/Code to reproduce bug**
In NVTabular 0.5.3:

```
training_set = nvt.Dataset("s3://path/to/parquet_files",
                        engine="parquet", part_size="200MB")
```
throws the following error:
```
FileNotFoundError                         Traceback (most recent call last)
<ipython-input-8-9efb33d625b3> in <module>
----> 1 train_set = nvt.Dataset("s3://path/to/parquet_files",
      2                         engine="parquet", part_size="200MB")

/nvtabular/nvtabular/io/dataset.py in __init__(self, path_or_source, engine, npartitions, part_size, part_mem_fraction, storage_options, dtypes, client, cpu, base_dataset, **kwargs)
    298             if isinstance(engine, str):
    299                 if engine == "parquet":
--> 300                     self.engine = ParquetDatasetEngine(
    301                         paths, part_size, storage_options=storage_options, cpu=self.cpu, **kwargs
    302                     )

/nvtabular/nvtabular/io/parquet.py in __init__(self, paths, part_size, storage_options, row_groups_per_part, legacy, batch_size, cpu)
     79                 else:
     80                     rg_byte_size_0 = _memory_usage(
---> 81                         cudf.io.read_parquet(f0, row_groups=0, row_group=0)
     82                     )
     83             row_groups_per_part = self.part_size / rg_byte_size_0

/opt/conda/lib/python3.8/site-packages/cudf/io/parquet.py in read_parquet(filepath_or_buffer, engine, columns, filters, row_groups, skiprows, num_rows, strings_to_categorical, use_pandas_metadata, *args, **kwargs)
    249 
    250     if engine == "cudf":
--> 251         return libparquet.read_parquet(
    252             filepaths_or_buffers,
    253             columns=columns,

cudf/_lib/parquet.pyx in cudf._lib.parquet.read_parquet()

cudf/_lib/parquet.pyx in cudf._lib.parquet.read_parquet()

cudf/_lib/io/utils.pyx in cudf._lib.io.utils.make_source_info()

FileNotFoundError: [Errno 2] No such file or directory: ['path/to/parquet_files/part-00000-15f94dcf-da4b-47f3-a72c-5f9372eb25c0-c000.snappy.parquet']
```

**Expected behavior**
This however works fine in NVTabular 0.5.2.

**Additional context**
I believe that somewhere in fsspec, the s3:// prefix is getting removed, which is causing the FileNotFoundError

**Comment by Arnie0426** [comment]
Found the reason - https://github.com/NVIDIA/NVTabular/commit/bc066c12949f6399ff3907005cacc6d95e2ed800 
Is the next version of Merlin expected to have the fix? Some of the new functionality in 0.5.3 is really cool (especially the dask-dataframe operations in the dataset class)

**Comment by benfred** [comment]
iirc, the problem was in our dependencies , and *should* be resolved with the next release.  I've re-enabled the test in #969 to verify.

We're targeting the end of the week for the next release, will keep you posted
